db/badgerdb/badger.go

```diff
@@ -0,0 +1,153 @@
+package badgerdb
+
+import (
+	"os"
+
+	"github.com/dgraph-io/badger/v3"
+	"go.vocdoni.io/dvote/db"
+)
+
+// MEMTABLESIZE defines the BadgerDB maximum size in bytes for memtable table.
+// The default is 64<<20 (64MB), this does not pre-allocate enough memory for
+// big Txs, that's why we use 128<<20.
+const MEMTABLESIZE = 128 << 20
```

Comment:
ALL_CAPS_CONSTANTS is a C/Java thing, in Go you should just go with MemTableSize :)